e2e/fixtures/events/discontinueClaim2v1Spec.js

```diff
@@ -108,6 +108,27 @@ module.exports = {
         };
       }
         break;
+      case 'ONE_V_ONE_NO_P_NEEDED': {
```

Comment:
this is not a problem with your ticket but i noticed the file name says 2v1 but here is says 1v1. maybe the file name needs changing?

Reply:
Thanks @sherlynkhaw , I'm going to change it